examples/vite-react/src/App.tsx

```diff
@@ -16,6 +18,7 @@ function App() {
         <img src={logo2} className="logo2" alt="logo2" />
         <p>Vite Example</p>
         <StyledText>This text is styled by styled-components</StyledText>
+        <p className={styles.green}>This text is styled by CSS Modules</p>
```

Comment:
We should add this for CRA example

Reply:
@ntt261298 Sure. Actually. I'm working on #41 and I was about to integrate it into this PR as well. But I think we can just update CRA example and put it to another PR. Will update CRA example.

Reply:
Update: I couldn't find a way to configure jest in `package.json` to work with CSS Modules. I think we can just leave it there, and wait for #41 to resolve this problem